cpp/ql/src/semmle/code/cpp/Class.qll

```diff
@@ -869,7 +912,17 @@ class TemplateClass extends Class {
 }
 
 /**
- * A class that is an instantiation of a template.
+ * A class that is an instantiation of a template.  For example in the following
+ * code there is a `MyTemplateClass<int>` instantiation:
```

Comment:
Perhaps add

For the `MyTemplateClass` template itself, see `TemplateClass`?

Reply:
Done.